docker/2.0.0/py3/Dockerfile.gpu

```diff
@@ -95,35 +98,43 @@ ENV PATH /usr/local/openmpi/bin/:$PATH
 ENV PATH=/usr/local/nvidia/bin:$PATH
 
 # SSH login fix. Otherwise user is kicked off after login
-RUN mkdir -p /var/run/sshd && sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+RUN mkdir -p /var/run/sshd \
+ && sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
 
 # Create SSH key.
-RUN mkdir -p /root/.ssh/ && \
-  ssh-keygen -q -t rsa -N '' -f /root/.ssh/id_rsa && \
-  cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys && \
-  printf "Host *\n  StrictHostKeyChecking no\n" >> /root/.ssh/config
+RUN mkdir -p /root/.ssh/ \
+ && ssh-keygen -q -t rsa -N '' -f /root/.ssh/id_rsa \
+ && cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys \
+ && printf "Host *\n  StrictHostKeyChecking no\n" >> /root/.ssh/config
 
 ###########################################################################
 # Python won’t try to write .pyc or .pyo files on the import of source modules
-ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1 PYTHONIOENCODING=UTF-8 LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV PYTHONDONTWRITEBYTECODE=1 
+ENV PYTHONUNBUFFERED=1 
+ENV PYTHONIOENCODING=UTF-8 
+ENV LANG=C.UTF-8 
+ENV LC_ALL=C.UTF-8
 
 WORKDIR /
 
-# before whl publish
-ARG sagemaker_tensorflow_extensions=tensorflow_gpu-2.0.0-cp36-cp36m-manylinux2010_x86_64.whl
-# ARG TF_URL="https://tensorflow-aws.s3-us-west-2.amazonaws.com/1.14/AmazonLinux/gpu/final/tensorflow-1.14.0-cp36-cp36m-linux_x86_64.whl"
-
-RUN ${PIP} --no-cache-dir install --upgrade pip setuptools
+RUN ${PIP} --no-cache-dir install --upgrade \
+    pip \
+    setuptools
 
 # Some TF tools expect a "python" binary
-RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
+ && ln -s $(which ${PIP}) /usr/bin/pip
 
-ARG framework_support_installable=sagemaker_tensorflow_container-2.0.8.dev0.tar.gz
-COPY $framework_support_installable .
-COPY $sagemaker_tensorflow_extensions .
+# Use THENSORFLOW_WHL instead of TF_URL before releasing
+# ARG TF_URL="https://tensorflow-aws.s3-us-west-2.amazonaws.com/1.14/AmazonLinux/gpu/final/tensorflow-1.14.0-cp36-cp36m-linux_x86_64.whl"
+
+ARG FRAMEWORK_SUPPORT_INSTALLABLE=sagemaker_tensorflow_container-2.0.8.dev0.tar.gz
```

Comment:
Define ARGs and ENVs at the top if they don't need to be defined/updated only after installation of a package.

Reply:
moved